website/client/components/static/staticWrapper.vue

```diff
@@ -1,14 +1,14 @@
 <template lang='pug'>
   div
-    static-header(:class='{"home-header": $route.name === "home"}')
+    static-header(v-if='$route.name !== "news"', :class='{"home-header": $route.name === "home"}')
```

Comment:
Can we make this a computed property `shouldHideContentWrap`? Or something similar

Reply:
makes sense! One moment